Hearthstone Deck Tracker/MainWindow/MainWindow_Edit.cs

```diff
@@ -213,6 +218,8 @@ public void ArchiveDeck(Deck deck, bool archive, bool saveAndUpdate = true)
 		internal async void BtnCloneSelectedVersion_Click(object sender, RoutedEventArgs e)
 		{
 			var deck = DeckPickerList.SelectedDecks.FirstOrDefault();
+            		deck = deck.GetSelectedDeckVersion();
```

Comment:
I see a nullreference exception waiting to happen here. Should probably be moved two lines down.